task/buildah/0.2/buildah.yaml

```diff
@@ -239,11 +247,28 @@ spec:
           sed -e '/^#/d' -e '/^\s*$/d' "${SOURCE_CODE_DIR}/${BUILD_ARGS_FILE}"
         )
       fi
-      # Append BUILD_ARGS
-      # Note: this may result in multiple --build-arg=KEY=value flags with the same KEY being
-      # passed to buildah. In that case, the *last* occurrence takes precedence. This is why
-      # we append BUILD_ARGS after the content of the BUILD_ARGS_FILE - they take precedence.
-      build_args+=("$@")
+
+      LABELS=()
+      # Split `args` into two sets of arguments.
```

Comment:
It looks complicated.
Could we have additional labels in a dedicated env var fro example?

Reply:
I started that way @mmorhun, but if we make the labels available to the script as an env var, then I don't think I can make the `param` an `array`.

Reply:
Will a delimiter in the env work? Like `label1=value1|label2=something else`